### PR TITLE
[Frontend] Integrated the Get Team API With Frontend Fixed

### LINKED ISSUE #724: [Frontend] Integrated the Get Team API With Frontend
Deliverables-

- [ ] The founder &Co-founder details should be hardcoded
- [ ] Rest all the cards should be populated a/c the data fetched from backend

Get Board Members API - https://github.com/HITK-TECH-Community/Community-Website/blob/main/backend/app/routes/teamMember/getTeamMembers.js

### PR DESCRIPTION
**Pull request by Hemu21**
## Issue that this pull request solves

[Issue Link](https://github.com/HITK-TECH-Community/Community-Website/issues/724) resolve #724 
 Closes: #724 

### Brief description of what is fixed or changed

The founder &Co-founder details is hardcoded and Rest all the cards are populated a/c the data fetched from backend. Checked user to redirect to respective social media account profile page. made default profile if image is not uploaded. 

## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] My changes does not break the current system and it passes all the current test cases.

### CODE REVIEW

**Comment by Hemu21** [comment]
@Kajol-Kumari  If you got free time can you please check and merge it.

File: frontend/src/pages/About/About.jsx [hunk 1]
```diff
@@ -20,6 +20,26 @@ const useStyles = makeStyles(() => ({
...
+      const response = await fetch(url);
+      const data = await response.json();
+      const _data = data.map((item) => {
+        return {
+          ...item,
+          teams: item.teams[0].split(","),
+        };
+      });
+      setTeam(_data);
+    } catch (error) {
```
Comment (Kajol-Kumari) [inline frontend/src/pages/About/About.jsx:39]: can we please a generic toast message for successful fetch/error?

File: frontend/src/pages/About/About.jsx [hunk 2]
```diff
@@ -203,27 +222,30 @@ export const About = (props) => {
                       <img
                         alt="profile"
                         className={style["cover"]}
-                        src={roleObject.profile_pic}
+                        src={roleObject.image||"./images/defaultUser.png"}
```
Comment (Kajol-Kumari) [inline frontend/src/pages/About/About.jsx:225]: image doesn't seem to be appearing, can u check once?

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
in founder and co-founder section can u keep a single card and add my info there, it should be hardcoded as it won't ever change, you take all the info from https://github.com/Kajol-Kumari

**Comment by Hemu21** [comment]
@Kajol-Kumari  Ok I will update

**Comment by Hemu21** [comment]
@Kajol-Kumari  I had updated it can you please check it.

File: frontend/src/pages/About/About.jsx (hunk 1 above)
Comment (Hemu21) [reply frontend/src/pages/About/About.jsx:67]: Done

**Review by Kajol-Kumari — APPROVED** [review]

**Pull request merged**